src/Avatar/Avatar.js

```diff
@@ -0,0 +1,185 @@
+/**
+ * Copyright 2017 CA
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+import React, { Children, cloneElement } from 'react';
+import { createStyledComponent, getNormalizedValue, pxToEm } from '../styles';
+import colors from '../colors';
+
+type Props = {
+  /** When `children` is a string, this will be the rendered text */
+  abbr?: string,
+  /**
+   * `img` (with an `alt` attribute), [Icon](../icon) (with a `title`), or a
+   * string
+   */
+  children: React$Node,
+  /** Background color */
+  color?:
+    | 'blue'
+    | 'dusk'
+    | 'gray'
+    | 'green'
+    | 'indigo'
+    | 'lime'
+    | 'magenta'
+    | 'orange'
+    | 'purple'
+    | 'red'
+    | 'sky'
+    | 'slate'
+    | 'teal'
+    | 'theme'
+    | 'yellow',
```

Comment:
I wish there was a way to have this list in one place. is there any way to pull this from [Color/index.js](https://github.com/mineral-ui/mineral-ui/blob/491-card-supplementals/src/website/app/pages/Color/index.js#L38)?

Reply:
@coldpour It is possible to import/export Flow types, but we haven't adopted that pattern yet in Mineral.  We started off this project with a very light touch in regards to Flow, after the painful experience with CAAC.  We'll get there before long I'm sure...